shoop/simple_supplier/templates/shoop/simple_supplier/admin/product_form_part.jinja

```diff
@@ -47,7 +47,7 @@
 {% endmacro %}
 
 {% call content_block(_("Stock management"), "fa-cubes") %}
-    {% if ss_form.can_manage_stock() %}
+    {% if ss_form.product.pk and ss_form.can_manage_stock() %}
```

Comment:
Instead of doing this you should probably skip the stock management tab all together when product is in new mode. So just yield this form in ``SimpleSupplierFormPart`` only when not in new mode.